### DISCUSSION TITLE
Software Scanner Crypto Mini App

### DISCUSSION THREAD

**Discussion by Cainismo** [rubenlagus/TelegramBots#1469] [Q&A]
Hello everyone! I wanted your help if someone can enlighten me with some answers towards the legitimacy of this mini app which claims to find Crypto Wallets that were lost from a specific timeframe and that you can withdraw money from them. As of now Im not seeing anything sketchy about it but your help would be MUCHH appreciated it. I will leave a link below of the Telegram MiniApp

https://t.me/SoftWareScannerS

![image](https://github.com/user-attachments/assets/7ee005b5-fd2e-48ba-9267-27c97e6f311c)

**Comment by Chase22** [comment]
99.9% it's a scam. But this repository is for building bots. We don't give support for third party bots

**Reply by Cainismo** [reply]
I didnt know where to ask. One of my close friends was about to pay money thats why i had to ask somewhere and see if it is legit. Sorry for posting in the wrong place.
